corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -50,15 +50,21 @@
                 });
             };
 
+            var tree_model = new LocationTreeViewModel(hierarchy);
+
             var reloadLocationSearchSelect = function() {
                 $('#location_search_select').select2('val', null);
                 enableLocationSearchSelect();
             };
 
+            var clearLocationSelection = function() {
+                reloadLocationSearchSelect();
+                tree_model.load(locs);
+            };
+
             enableLocationSearchSelect();
```

Comment:
I think I missed this in a previous PR, but took me a while to understand this part because it looks like there's a simple indentation error that starts at the `dataType: 'json',` line above and ends here, making this look as if it's all inside `enableLocationSearchSelect`, whereas that function actually ends above on line 51.

It's only semi-related to this change, but since you're adding to this area, would you mind fixing this indentation in this PR as well?